app/models/type.rb

```diff
@@ -72,9 +72,9 @@ def <=>(type)
     name <=> type.name
   end
 
-  def self.all
-    find(:all, order: 'position')
-  end
+  # def self.all
+  #  find(:all, order: 'position')
+  # end
```

Comment:
> Why do you comment out your code if you have git available? ;-)

@NobodysNightmare this was done many months ago while I was trying to get the app to boot. During the "spiking" phase, I tend to do things less elegantly than I would normally.